lib/rbtrace/cli.rb

```diff
@@ -407,6 +415,10 @@ def self.run
           tracer.puts "=> #{res}"
         end
 
+      elsif opts[:interactive_given]
+        require "rbtrace/interactive/#{opts[:interactive]}"
+        load `which #{opts[:interactive]}`.chomp
```

Comment:
This is probably not any better, but I do still want to provide alternatives:
https://github.com/tmm1/rbtrace/pull/39/commits/7b441dd1258c978f56c6b765735e2c545e17838b

Please let me know what do you think. Also, I don't know why, but using `irb` would always timeout and show `*** timed out waiting for eval response` but it would still show the correct result. However, `rib` doesn't have this issue. Maybe it's because the way `irb` was implemented. Some others could try to use this on `pry`.